Python/compile.c

```diff
@@ -2190,12 +2186,9 @@ compiler_function_body(struct compiler *c, stmt_ty s, int is_async, Py_ssize_t f
             docstring = NULL;
         }
     }
-    if (compiler_add_const(c, docstring ? docstring : Py_None) < 0) {
-        Py_XDECREF(docstring);
-        compiler_exit_scope(c);
-        return ERROR;
-    }
-    Py_CLEAR(docstring);
+    int ret = compiler_add_const(c, docstring ? docstring : Py_None);
```

Comment:
```suggestion
    int ret = (int)compiler_add_const(c, docstring ? docstring : Py_None);
```

Just to avoid compiler warnings. Or you could have a more verbose `compiler(...) < 0 ? ERROR : SUCCESS` to avoid the downcast.